pkg/controller/importconfig/manifests/klusterlet/klusterlet.yaml

```diff
@@ -11,11 +11,12 @@ spec:
 {{- if eq .InstallMode "Hosted"}}
     mode: Hosted
 {{- else }}
-    mode: Default
+    mode: Singleton
 {{- end}}
   registrationImagePullSpec: "{{ .RegistrationImageName }}"
   workImagePullSpec: "{{ .WorkImageName }}"
   clusterName: "{{ .ManagedClusterNamespace }}"
+  imagePullSpec: "{{ .ImageName }}"
```

Comment:
How about using `imagePullSepc: "{{ . RegistrationOperatorImage}}"`, it's the same value.